nowcasting_dataset/data_sources/data_source.py

```diff
@@ -210,9 +211,9 @@ def create_batches(
 
             # Generate batch.
             batch = self.get_batch(
-                t0_datetimes=locations_for_batch.t0_datetime_UTC,
-                x_locations=locations_for_batch.x_center_OSGB,
-                y_locations=locations_for_batch.y_center_OSGB,
+                t0_datetimes_utc=locations_for_batch.t0_datetime_UTC,
+                x_meters_osgb=locations_for_batch.x_center_OSGB,
```

Comment:
ah, here and elsewhere, I'd suggest replacing `{x,y}_meters_osgb` with `{x,y}_center_osgb`.  I think the word "center" is probably more important than the word "meters", here, because OSGB is always in meters.

Reply:
Thanks, Ive no run a search through the library, and changed any 'x_meters' to 'x_center'

Reply:
Great, thank you!